### ISSUE TITLE
Integration not loading since 1.9.4 (cannot import name 'AlarmControlPanelEntityFeature')

### ISSUE THREAD

**Issue by bcutter** [nielsfaber/alarmo#495]
### Alarmo Version

v1.9.4

### HA Version

2022.4.7

### Bug description

After updating to v1.9.4 (using HACS) after restarting HA there's a log entry and Alarmo is not loading:

![grafik](https://user-images.githubusercontent.com/13799156/172026820-0da51702-521f-4766-b940-3832ede3b128.png)

### Steps to reproduce

1) Update to Alarmo v1.9.4 (using HACS)
2) Restart HA

### Relevant log output

```shell
Logger: homeassistant.setup
Source: setup.py:162
First occurred: 4. Juni 2022, 23:52:13 (1 occurrences)
Last logged: 4. Juni 2022, 23:52:13

Setup failed for custom integration alarmo: Unable to import component: cannot import name 'AlarmControlPanelEntityFeature' from 'homeassistant.components.alarm_control_panel' (/usr/src/homeassistant/homeassistant/components/alarm_control_panel/__init__.py)
```
```

**Comment by nielsfaber** [comment]
Your version of HA is incompatible with the latest version of Alarmo. The 1.9.4 release contains several changes to work with HA 2022.6.
I cannot make things backwards compatible unfortunately.

**Comment by bcutter** [comment]
I meanwhile upgraded to 2022.5.5 and the issue is gone.

In general: I think there are mechanisms of setting a minimum needed version of HA in HACS so these incompatible version constellations don't arise.

**Comment by nielsfaber** [comment]
In release [1.8.6](https://github.com/nielsfaber/alarmo/releases/tag/v1.8.6) I experimented with this setting.
I defined in the HACS config file that 2022.3 or newer was required for this release: https://github.com/nielsfaber/alarmo/blob/v1.8.6/hacs.json.

The result was that for all users on other HA version than 2022.3, the Alarmo integration was completely hidden from HACS.
They would also not see 1.8.5 as installation or update candidate.
My conclusion was that HACS didn't provide the desired functionality, so ever since I stayed away from this setting.
But I would also prefer this to work as it should, I don't want to force you to upgrade HA (since it usually cost an hour of your weekend).
